lib/smart_proxy_openscap/arf_parser.rb

```diff
@@ -87,7 +87,8 @@ def populate_fixes(fixes, set_values)
           {
             :id => fix.id,
             :system => fix.system,
-            :full_text => fix.full_text(set_values)
+            :full_text => fix.full_text(set_values),
+            :reboot => fix.instance_variable_get('@parsed_xml')['reboot'] # TODO: add this to openscap_parser lib
```

Comment:
Well, it does the trick. Could we get at least an issue to remove this and ideally in openscap_parser to expose this?

Reply:
Done: https://projects.theforeman.org/issues/36825.

Although, it may take some time until we will do that. If I'm not mistaken, the lib now requires Ruby 3+.

Reply:
That's fine, as long as it won't get lost.

Reply:
That's what `TODO`s for :angel: